integration_tests/src/main/python/orc_test.py

```diff
@@ -486,4 +486,47 @@ def test_orc_read_with_corrupt_files(spark_tmp_path, reader_confs, v1_enabled_li
 
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : spark.read.orc([first_data_path, second_data_path, third_data_path]),
-            conf=all_confs)
\ No newline at end of file
+            conf=all_confs)
+
+conf_for_orc_aggregate_pushdown = {
+    "spark.sql.orc.aggregatePushdown": "true", 
+    "spark.sql.sources.useV1SourceList": ""
+}
+
+@pytest.mark.skipif(is_before_spark_330(), reason='Aggregate push down on ORC is a new feature of Spark 330')
+@allow_non_gpu(any = True)
+def test_orc_scan_with_aggregation_pushdown_fallback(spark_tmp_path):
+    """
+    The aggregation will be pushed down in this test, so we should fallback to CPU
+    """
+    data_path = spark_tmp_path + '/pushdown.orc'
+
+    def do_orc_scan(spark):
+        df = spark.read.orc(data_path).selectExpr("count(p)")
```

Comment:
I don't see a test for the `Min` aggregation pushdown fallback.  Also these tests are very similar, so I suggest using `pytest.parametrize` to parameterize the test based on the operation and then we can succinctly cover all of the operations in one test, reusing all the boilerplate setup and assertion code that is common across the operator dimension.